### DISCUSSION TITLE
Import form entries

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco.Forms.Issues#1520] [Ideas]
It is possible to export form entries in Umbraco Forms.

![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/2919859/bf8c7df2-9bf4-42d6-8757-3cbabc84a368)

while migrating a project from Umbraco 8 to Umbraco 12, I noticed when Forms are transferred, the submitted entries are not transferred.

It isn't crucial for this specific project, but I wonder it is should be possible to import as well? I a setting which always transferring/restoring form entries as well?

**Comment by TvGessel** [comment]
While the migration should technically work, we haven't specifically tested the path from v8 to v12. Since the import isn't a common generic use case, we don't have immediate plans to realize this.
